debian/snapd.postrm

```diff
@@ -13,15 +13,25 @@ systemctl_stop() {
 if [ "$1" = "purge" ]; then
     mounts=$(systemctl list-unit-files | grep '^snap[-.].*\.mount' | cut -f1 -d ' ')
     services=$(systemctl list-unit-files | grep '^snap[-.].*\.service' | cut -f1 -d ' ')
-    for unit in $services $mounts; do
-        # ensure its really a snapp mount unit or systemd unit
+
+    for unit in $mounts; do
+        # ensure its really a snap mount unit or systemd unit
```

Comment:
Why is this changing? It's unmounting the underlying filesystem from the services before it stops the services? The original logic handled both in the same loop, accounting for what each one is. Not clear why we need to split it up.